common/aws_io_params.yaml

```diff
@@ -588,3 +588,47 @@ i7ie.ALL:
   read_bandwidth: 3422623232
   write_iops: 119327
   write_bandwidth: 1526442410
+i8g.large:
+  read_iops: 82268
+  read_bandwidth: 605359957
+  write_iops: 45598
+  write_bandwidth: 419360096
+i8g.xlarge:
+  read_iops: 164442
+  read_bandwidth: 1216944170
+  write_iops: 90446
+  write_bandwidth: 838613674
+i8g.2xlarge:
+  read_iops: 328275
+  read_bandwidth: 2438557696
+  write_iops: 134545
+  write_bandwidth: 1687947733
+i8g.4xlarge:
+  read_iops: 553607
+  read_bandwidth: 4797597013
+  write_iops: 140814
+  write_bandwidth: 3419948373
+i8g.8xlarge:
+  read_iops: 527429
+  read_bandwidth: 4797424981
+  write_iops: 139669
+  write_bandwidth: 3421133482
```

Comment:
it's the same as i8g.4xlarge, while all previous ones scaled quite linearly.

Reply:
I see that it's the top for write starting 4xlarge and above.
Does it make sense?

Reply:
Yes, it seems like i8g.4xlarge and above are same performance, when measuring per single disk (number of disks will increase when instance size become larger, so RAID0 performance probably will increase).
(see detail of the result at https://github.com/scylladb/scylla-machine-image/pull/676#issue-2933021771)

Maybe we should make these entries into one, something like (we have such virtual instance size named "ALL"):
```
i8g.ALL:
  read_iops: 553607
  read_bandwidth: 4797597013
  write_iops: 140814
  write_bandwidth: 3419948373
```

Reply:
@avikivity does it make sense to you?
It seems like it would be better to use multiple i8g.4x than scale up to i8g.8x and above.

Reply:
Maybe it's limited by PCI bandwidth.

We can retest with fio to increase confidence.

Reply:
/cc @xemul, also see the published AWS numbers below.